### PR TITLE
Add more unit tests around NamedModelViewSet

### PR DESCRIPTION
**Pull request by dralley**
closes #3567
https://pulp.plan.io/issues/3568

### CODE REVIEW

File: pulpcore/tests/viewsets/test_base.py [hunk 1]
```diff
@@ -1,34 +1,222 @@
...
+from django.http import Http404
 from django.test import TestCase
+from rest_framework.serializers import ValidationError as DRFValidationError
 
-from pulpcore.app import models, viewsets
+from pulpcore.app import models, viewsets, serializers
 
 
 class TestGetQuerySet(TestCase):
     def test_adds_filters(self):
```
Comment (dralley) [inline pulpcore/tests/viewsets/test_base.py]: This test currently fails and I'm not sure why.  ```queryset``` is empty for some reason.

**Comment by dralley** [comment]
@daviddavis should I disable the test for now?  As I mentioned, it was not *actually* testing anything previously.

**Comment by daviddavis** [comment]
It would be a nice test to have but if you don't have the time or can't figure it out, then +1.

**Comment by dralley** [comment]
I'll disable it for now.  We can figure it out when we do some unit testing passes later on.

**Pull request merged**